filecache_test.go

```diff
@@ -276,4 +283,42 @@ var _ = Describe("Filecache", func() {
 			Expect(dir2).To(Equal("dc"))
 		})
 	})
+
+	Describe("NewDownloadRecord()", func() {
```

Comment:
This replaces the Convey tests that I removed from Lazyraster [here](https://github.com/Nitro/lazyraster/pull/47/files#diff-7d77b508b05298b61ba32ceb5bfd80a6L42)